druid-shell/src/platform/x11/application.rs

```diff
@@ -249,6 +308,41 @@ impl Application {
                     self.finalize_quit();
                 }
             }
+            Event::ConfigureNotify(ev) => {
```

Comment:
Like the above comment says, there needs to be a check against `self.window_id` because this event could also fire for the app `InputOnly` window.